resources/automagic_dashboards/orders.yaml

```diff
@@ -0,0 +1,107 @@
+bindings:
+  fields:
+  - named:
```

Comment:
we shouldn't be hard coding names in this.  this is the kind of stuff that should be in the metadata system. If it's a type, then the user can manually add that. if it's a name, then not only is it very fragile, but it's hopeless for non-english. 

Also, I don't necessarily understand why we're using a binding vs just having the heuristic specify a type in-line. This seems needlessly complicated. It's asking analysts to declare variables which is ... not quite going to happen,